docs/source/changelog/0.13.0-changelog.rst

```diff
@@ -0,0 +1,199 @@
+*******
+v0.13.0
+*******
+
+:Date: December 02, 2021
+
+Contributors
+============
+
+A total of 25 people contributed to this
+release. People with a '+' by their names authored a patch for the first
+time.
+
+* Benjamin Hackl
+* Christopher Besch
+* Darylgolden
+* Filip +
+* Laith Bahodi
+* Lucas Ricci +
+* ManimBot +
```

Comment:
do we want to include the manimbot?

Reply:
I guess no, we should ignore like the pre-commit bot

Reply:
I've adapted the changelog script.